src/components/Auth/index.js

```diff
@@ -0,0 +1,90 @@
+require('dotenv').config();
+const bcrypt = require('bcrypt');
+const generateTokens = require('./helper');
+const User = require('../User/model');
+const UserService = require('../User/service');
+
+const {
+    signUpBodyValidation,
+    logInBodyValidation,
+} = require('./validation');
+
+/**
+ * @function
+ * @param {express.Request} req
+ * @param {express.Response} res
+ * @param {express.NextFunction} next
+ * @returns {Promise < void >}
+ */
+async function signUp(req, res, next) {
+    try {
+        const { error, value } = signUpBodyValidation(req.body);
+        if (error) {
+            return res
+                .status(400)
+                .json({ error: true, message: error.details[0].message });
```

Comment:
please send array of messages to client. 
error key must contain error code or error name - not boolean value.